providers/src/airflow/providers/google/marketing_platform/operators/analytics.py

```diff
@@ -1,551 +0,0 @@
-
-    :param account_id: The GA account Id (long) to which the data upload belongs.
-    :param web_property_id: The web property UA-string associated with the upload.
-    :param custom_data_source_id: The id to which the data import belongs.
-    :param gcp_conn_id: The connection ID to use when fetching connection info.
-    :param delegate_to: The account to impersonate using domain-wide delegation of authority,
-        if any. For this to work, the service account making the request must have
-        domain-wide delegation enabled.
-    :param api_version: The version of the api that will be requested for example 'v3'.
-    :param impersonation_chain: Optional service account to impersonate using short-term
-        credentials, or chained list of accounts required to get the access_token
-        of the last account in the list, which will be impersonated in the request.
-        If set as a string, the account must grant the originating account
-        the Service Account Token Creator IAM role.
-        If set as a sequence, the identities from the list must grant
-        Service Account Token Creator IAM role to the directly preceding identity, with first
-        account from the list granting this role to the originating account (templated).
-    """
-
-    template_fields: Sequence[str] = ("impersonation_chain",)
-
-    def __init__(
-        self,
-        account_id: str,
-        web_property_id: str,
-        custom_data_source_id: str,
-        gcp_conn_id: str = "google_cloud_default",
-        delegate_to: str | None = None,
-        api_version: str = "v3",
-        impersonation_chain: str | Sequence[str] | None = None,
-        **kwargs,
-    ) -> None:
-        super().__init__(**kwargs)
-
-        self.account_id = account_id
-        self.web_property_id = web_property_id
-        self.custom_data_source_id = custom_data_source_id
-        self.gcp_conn_id = gcp_conn_id
-        self.delegate_to = delegate_to
-        self.api_version = api_version
-        self.impersonation_chain = impersonation_chain
-
-    def execute(self, context: Context) -> None:
-        ga_hook = GoogleAnalyticsHook(
-            gcp_conn_id=self.gcp_conn_id,
-            delegate_to=self.delegate_to,
-            api_version=self.api_version,
-            impersonation_chain=self.impersonation_chain,
-        )
-
-        uploads = ga_hook.list_uploads(
-            account_id=self.account_id,
-            web_property_id=self.web_property_id,
-            custom_data_source_id=self.custom_data_source_id,
-        )
-
-        cids = [upload["id"] for upload in uploads]
-        delete_request_body = {"customDataImportUids": cids}
-
-        ga_hook.delete_upload_data(
-            self.account_id,
-            self.web_property_id,
-            self.custom_data_source_id,
-            delete_request_body,
-        )
-
-
-class GoogleAnalyticsModifyFileHeadersDataImportOperator(BaseOperator):
```

Comment:
@moiseenkov  should we remove this operator, because we do not have a `deprecate` message here?

Reply:
Strictly peaking we should, but I decided to remove it because this operator doesn't bring any value and doesn't cover any relevant use case without old Google Analytics operators. Basically, it was implemented only for  adjusting data format before importing it to Google Analytics. And since the API is shut down, there is no need to reformat data, because the data import is impossible anymore.